src/pages/home.js

```diff
@@ -19,17 +22,81 @@ var Impact = React.createClass({
   }
 });
 
+const DONATE_CTA_DELAY = 750; // in milliseconds
+
 var Signup = React.createClass({
   contextTypes: {
     intl: React.PropTypes.object
   },
+  getInitialState: function() {
+    return {
+      showModal: false,
+      dismissedModal: false
+    };
+  },
+  componentDidMount: function() {
+    if (typeof window !== "undefined" && window.addEventListener) {
+      this._withScroll = e => this.handleScroll(e);
+      window.addEventListener('scroll', this._withScroll);
+    }
+  },
+  componentWillUnmount() {
+    if (this._withScroll) {
+      window.removeEventListener('scroll', this._withScroll);
+    }
+  },
+  handleScroll: function(e) {
+    if (this._ctaTimeout) {
+      clearTimeout(this._ctaTimeout);
+    }
+    this._ctaTimeout = setTimeout(() => this.spawnDonateCTA(), DONATE_CTA_DELAY);
+  },
+  spawnDonateCTA: function() {
+    if (!storage.getItem('dismissedModal')) {
+      this.setState({
+        showModal: true
+      });
+    }
+  },
+  closeModal: function() {
+    this.setState({
+      showModal: false
+    }, () => storage.setItem('dismissedModal', 'true'));
+  },
+  generateModal: function() {
+    if (this.state.showModal) {
+      return this.generateDonationModal();
+    }
+    return null;
+  },
+  generateDonationModal: function() {
+    return (
+      <Modal onClose={() => this.closeModal()}>
+        <section className="donate-container">
+          <h2>We all love the web.<br/> Join Mozilla in defending it.</h2>
```

Comment:
This website is localized in `de, es, fr, it, pl`, do you mind extracting the strings here? https://github.com/mozilla/copyright/blob/master/locales/en-US/copyright.properties#L36

Reply:
ping @gthomas to make sure this is indeed the text we want.